syntax = "proto3";

service Game_Over{
    rpc game_over (game_over_req) returns (game_over_response){}
}

message game_over_req
{

}

message game_over_response
{
    bool client_game_over = 1;
}